contracts/governance/extensions/GovernorTimelockCompound.sol

```diff
@@ -24,7 +24,7 @@ abstract contract GovernorTimelockCompound is IGovernorTimelock, Governor {
     ICompoundTimelock private _timelock;
 
     /// @custom:oz-retyped-from mapping(uint256 => GovernorTimelockCompound.ProposalTimelock)
```

Comment:
We don't need this anymore after 5.0, right? Non-blocking and we shouldn't remove it yet but I think we may need to create a ticket for removing these. CC @frangio